probe_scraper/runner.py

```diff
@@ -20,7 +20,7 @@
 from .parsers.metrics import GleanMetricsParser
 from .parsers.repositories import RepositoriesParser
 from .parsers.scalars import ScalarsParser
-from .scrapers import git_scraper, moz_central_scraper
+from .scrapers import dependency_scraper, git_scraper, moz_central_scraper
```

Comment:
Where is `dependency_scraper` implemented?

Reply:
Gah -- forgot to `git add` it.  Should be here now.